gui.lua

```diff
@@ -75,60 +75,58 @@ function awards.get_formspec(name, to, sid)
 		formspec = formspec .. "button_exit[4.2,2.3;3,1;close;"..minetest.formspec_escape(S("OK")).."]"
 		return formspec
 	end
+	sid = awards_list[sid] and sid or 1
 
 	-- Sidebar
-	if sid then
```

Comment:
From here on: Only removed this condition and replaced `item` with `sitem` (see definition below) where it was needed.